packages/react-components/src/components/DevicePermissions/DomainPermissions.tsx

```diff
@@ -23,6 +23,10 @@ export interface CommonDomainPermissionsProps {
    * Type of the Domain Permissions component.
    */
   type: 'request' | 'denied' | 'check';
+  /**
+   * Type of the browser domain permission component is rendered in.
+   */
+  browser?: string;
```

Comment:
browser can't be `string` here - how would contoso know what string value to provide, and what will happen with specific string values?
Consider `browser: 'safari' | 'other' with @defaultValue other`

Reply:
updated